tools/hlstool/hlstool.cpp

```diff
@@ -119,40 +119,67 @@ static cl::opt<HLSFlow>
                                   "Statically scheduled (Calyx path)")),
             cl::cat(mainCategory));
 
-enum DynamicParallelismKind {
-  DynamicParallelismNone,
-  DynamicParallelismLocking,
-  DynamicParallelismPipelining
-};
+enum DynamicParallelismKind { None, Locking, Pipelining };
```

Comment:
I'd prefer this to be an enum class to avoid bugs related to naming / automatic conversions.